src/reanimated2/animation/springUtils.ts

```diff
@@ -84,6 +87,11 @@ export function validateConfig(config: DefaultSpringConfig): boolean {
     errorLog += `duration can't be negative, got ${config.duration}, `;
   }
 
+  if (config.clamp && config.clamp[0] > config.clamp[1]) {
+    configIsInvalid = true;
+    errorLog += `clamp should have format [number1, number2], where number1 is smaller than number2, got [${config.clamp[0]}, ${config.clamp[1]}]`;
```

Comment:
This error message sounds too detailed, let's just tell the user that the lower bound should be not greater than the upper bound.